lib/driver.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'csv_record'
+
+module RideShare
+  class Driver < CsvRecord
+    attr_reader :id, :name, :vin, :trips
+    attr_accessor :status
+
+    def initialize(id:, name:, vin:, status: :AVAILABLE, trips: [])
+      super(id)
+
+      @name = name
+      @vin = vin
+      raise ArgumentError.new("Invalid VIN") if (vin.length) != 17
```

Comment:
It's helpful for debugging to include the bad argument in your variable:

```suggestion
      raise ArgumentError.new("Invalid VIN:  #{vin}") if (vin.length) != 17
```